content/en/docs/ops/configuration/security/security-policy-examples/index.md

```diff
@@ -0,0 +1,197 @@
+requests without JWT token.
+
+After defining the `RequestAuthentication` policy, use the following extra policy to enforce mandatory JWT validation
+that rejects the request if it has no JWT token:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: require-jwt
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  action: DENY
+  rules:
+  - from:
+    - source:
+        notRequestPrincipals: ["*"]
+EOF
+{{< /text >}}
+
+### Require different JWT issuer per host
+
+JWT validation is common on the ingress gateway and you may want to require different JWT issuers for different
+hosts, you can use the authorization policy for finer grained JWT validation in addition to the request authentication.
+
+Use the following policy if you want to allow requests only to the listed hosts and reject requests to any other hosts:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: jwt-per-host
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        hosts: ["example.com", "*.example.com"]
+        # the JWT token must have issuer with suffix "@example.com"
+        requestPrincipals: ["*@example.com"]
+    - source:
+        hosts: [".another.org", "*.another.org"]
+        # the JWT token must have issuer with suffix "@another.org"
+        requestPrincipals: ["*@another.org"]
+{{< /text >}}
+
+Alternatively, use the following policy if you want to require JWT validation only to the listed hosts but not for any
+other hosts:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: jwt-per-host
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  action: DENY
+  rules:
+  - from:
+    - source:
+        hosts: ["example.com", "*.example.com"]
+        # the JWT token must have issuer with suffix "@example.com"
+        notRequestPrincipals: ["*@example.com"]
+    - source:
+        hosts: [".another.org", "*.another.org"]
+        # the JWT token must have issuer with suffix "@another.org"
+        notRequestPrincipals: ["*@another.org"]
+{{< /text >}}
+
+### Namespace isolation
+
+You want to block all traffic from outside the namespace `foo`, in other words, isolate the namespace `foo` from other
+namespaces. This requires to enable mTLS first.
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: foo-isolation
+  namespace: foo
+spec:
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        namespaces: ["foo"]
+{{< /text >}}
+
+If you want to make sure the policy can not be bypassed by other `ALLOW` policies, you can use the `DENY` policy to
+achieve the result:
```

Comment:
Do we have to give an example DENY policy for every use case? I feel it is a little repeating.

Reply:
Yeah, that's good suggestion. I think what I really want to show is the non bypassable property of the DENY policy. Moved the duplicate deny policies to a dedicated section to make this more clear.